cudax/include/cuda/experimental/__execution/bulk.cuh

```diff
@@ -152,7 +161,7 @@ struct _CCCL_TYPE_VISIBILITY_DEFAULT __bulk_t
       execution::set_stopped(static_cast<_Rcvr&&>(__state_->__rcvr_));
     }
 
-    [[nodiscard]] _CCCL_NODEBUG_API constexpr auto get_env() const noexcept -> __fwd_env_t<env_of_t<_Rcvr>>
```

Comment:
I do not understand the consequences of this macro change.

Reply:
@ericniebler You mean, it stops inlining.  We do need to restore this at some point : - )